src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,6 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
```

Comment:
This interface requires a header comment to allow other readers to better understand the purpose of the interface.